modules/ROOT/pages/java-embedded/setup.adoc

```diff
@@ -136,11 +136,10 @@ For maintainability purposes, you can define your embedded DBMS configuration in
 
 [source, properties]
 ----
-dbms.mode=CORE
-dbms.default_advertised_address=core01.example.com
-dbms.default_listen_address=0.0.0.0
-causal_clustering.discovery_type=LIST
-causal_clustering.initial_discovery_members=core01.example.com,core02.example.com,core03.example.com
+server.default_advertised_address=core01.example.com
+server.default_listen_address=0.0.0.0
+dbms.cluster.discovery.type=LIST
+dbms.cluster.discovery.endpoints=core01.example.com,core02.example.com,core03.example.com
 dbms.connector.bolt.enabled=true
 dbms.connector.http.enabled=true
```

Comment:
Just ran a server with this config, and it also complains about these two being deprecated, looks like since 5.0.
```suggestion
server.bolt.enabled=true
server.http.enabled=true
```